src/css/base/typography.scss

```diff
@@ -53,12 +61,10 @@ h5 {
   font-size: 1rem;
 }
 
-body {
-  font-weight: 300;
-  line-height: 1.7333333;
-}
-
 p {
+  font-family: $font-serif;
+  font-size: 1.05rem;
+  line-height: 1.75;
```

Comment:
When line-height doesn't have a unit, it makes it relative to the font-size of the current element, which is what I wanted.